src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X] " : "[ ] "); // mark done task with X
+    }
+
+    public String getDescription(){
+        return this.description;
+    }
+
+    public void markAsDone(){
```

Comment:
Could be made more efficient by passing a boolean parameter that can mark as true or mark as false based on the parameter.